client/reducers/foods.js

```diff
@@ -1,10 +1,19 @@
 import { GET_FOODS_SUCCESS } from './../actions/foods'
+import { GET_CATEGORIES_SUCCESS } from './../actions/categories'
 
 export default function foodReducer (state = [], action) {
   switch (action.type) {
     case GET_FOODS_SUCCESS:
       return action.foods
+    default:
+      return state
+  }
+}
 
+export default function categoryReducer (state = {}, action) {
```

Comment:
Is it not possible to have two `default` exports in one file. I suspect your intent was actually just to add to the `switch` statement the following:
```js
case GET_CATEGORY_SUCCESS:
  return action.category
```